modules/troubleshooting/pages/system-logs.adoc

```diff
@@ -1,10 +1,11 @@
-= Log files
+= System Logs
```

Comment:
The one thing that's not clear to me here is how does system log differ from audit logs? Or this page still sounds like it is giving an overview of all logs.